scripts/v.in.wfs/v.in.wfs.py

```diff
@@ -94,21 +117,37 @@
 from grass.script.utils import try_remove
 from grass.script import core as grass
 
-from urllib.request import urlopen
-from urllib.request import build_opener, install_opener
-from urllib.request import HTTPPasswordMgrWithDefaultRealm, HTTPBasicAuthHandler
-from urllib.error import URLError, HTTPError
+try:
+    from urllib2 import urlopen, URLError, HTTPError
```

Comment:
The `urllib2` lib is a Python 2 lib. We currently do not need Python 2 version support.